src/js/providers/html5.js

```diff
@@ -782,6 +790,45 @@ define([
         this.getName = function() {
             return { name : _name };
         };
+
+        this.getTextTracks = _getTextTracks;
+
+        //model expects setSubtitlesTrack when changing subtitle track
+        this.setSubtitlesTrack = _setSubtitlesTrack;
+
+        function _setTextTracks(tracks) {
+            //filter for tracks where kind = 'subtitles'
+
+            if(tracks && tracks.length > 0) {
+                _textTracks = _.filter(tracks, function(track) {
+                    return track.kind === 'subtitles';
+                });
+                _this.trigger('subtitlesTracks',{tracks: _textTracks});
+            }
+        }
+
+        function _getTextTracks() {
+            return _textTracks;
+        }
+
+        function _setSubtitlesTrack(index) {
+            //index off by 1 because of 'off' option
+            if(_currentTextTrackIndex !== undefined) {
```

Comment:
Can you make the default `-1` like `_currentQuality`? I'd like us to stick to assigning variables a value when their declared.